extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -192,6 +209,46 @@ oppia.directive('imageWithRegionsEditor', [
             height: (area[1][1] - area[0][1]) * $scope.getImageHeight()
           };
         };
+        var resizeRegion = function() {
+          var labeledRegions = $scope.$parent.value.labeledRegions;
+          var resizedRegion = labeledRegions[$scope.selectedRegion].region;
+          var deltaX = $scope.mouseX - $scope.originalMouseX;
+          var deltaY = $scope.mouseY - $scope.originalMouseY;
+          var x = $scope.originalRectArea.x;
+          var y = $scope.originalRectArea.y;
+          var width = $scope.originalRectArea.width;
+          var height = $scope.originalRectArea.height;
+          var newWidth = width - $scope.xDirection * deltaX;
+          var newHeight = height - $scope.yDirection * deltaY;
+          // Margin between the position mouse is pressed to resize and the y
```

Comment:
Comment is a bit unclear. Suggest rewording, e.g.:

The distance between where the mouse was first clicked to initiate the resize action, and the ...

(I don't know what "..." should be. y/x coordinate of what?)